lightwood/helpers/text.py

```diff
@@ -210,7 +210,12 @@ def get_identifier_description_mp(arg_tup):
 
 def get_identifier_description(data, column_name, data_dtype):
     data = list(data)
```

Comment:
why do we make this a list?

Reply:
if it's already in a series form you can simply do something like len(data.unique())